### ISSUE TITLE
[5pt] Update "generate_nws_lid.py" and generate new NWS lid points, including Alaska points

### ISSUE THREAD

**Issue by AliForghani-NOAA** [NOAA-OWP/inundation-mapping#1132]
Update "generate_nws_lid.py" and generate new NWS lid points, including Alaska points.
Also, need to compare new lid points (simply in QGIS) with previous ones. There may be a few changes (additions/removals) in the points for the CONUS domain but we mainly want to see the AK forecast points added -- need the updated lid points for AK, before we can generate catfim.

**Comment by mluck** [comment]
This might be a duplicate of #1130.

**Comment by AliForghani-NOAA** [comment]
This issue is about updating `tools/generate_nws_lid.py`. Does issue #1130 involve updating this file?

**Comment by mluck** [comment]
Ah, you're right.

**Comment by AliForghani-NOAA** [comment]
@CarsonPruitt-NOAA 
This map displays 4741 NWS_LID points compiled using `tools/generate_nws_lid.py`:
![image](https://github.com/NOAA-OWP/inundation-mapping/assets/124194814/42303aca-4631-43e8-97bc-70fd0974ddb2)

Here I am retrieving stations from these two groups:
- lid stations with `rfc_forecast_point= True` 
- lid stations in `/data/inputs/ahp_sites/evaluated_ahps_sites.csv`

The lid stations in AK (Alaska), HI, and PR, with above two criteria have also been selected, as shown in the map. In the previous version of the code, **all of**  lid stations in PR and HI (regardless of meeting above two criteria), were also being retrieved. I have updated this version to exclude such stations. 

Similar to 'usgs_gages.gpkg' dataset compiled last week, the selected lid stations, including those in Alaska, will be stored in a single gpkg file (`nws_lid.gpkg`) with EPSG=5070. The Alaska stations can be identified using their HUC8 numbers (beginning with '19'). 

Could you please confirm:
1- If the above two criteria adequately compile all necessary NWS_lid stations 
2- If it is correct to include all stations, especially Alaska stations, in a single gpkg file with EPSG=5070?

**Comment by CarsonPruitt-NOAA** [comment]
Thanks for the detailed review, Ali! Yes on both accounts. Let's move forward with what you have.

**Comment by AliForghani-NOAA** [comment]
Sounds Good!
